analytics/config/config.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import (
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/analytics/filesystem"
+	"github.com/prebid/prebid-server/config"
```

Comment:
This package's name is also "config" ... It does work, but may cause a little confusion. I don't know a better solution right now though.

Reply:
yeah... I'm in the same situation there. The purpose of the package would literally be "to build up the implementations based on the config"... I could call it `construction`? `instantiation`? `building`?

There are a bunch of packages which follow this format, and where this pattern could make sense (fetchers, caches, adapters, httprouting...), so let me know if you do come up with something you like more.